cmd/proxy/tcp_proxy.go

```diff
@@ -48,7 +48,8 @@ func (proxy *TCPProxy) clientLoop(client *net.TCPConn, quit chan bool) {
 				from.CloseWrite()
 			}
```

Comment:
Uh, thank you for the comment and sorry for I did not respond.. 😅

I will try to remove that block after some verification